src/sagemaker/workflow/pipeline.py

```diff
@@ -481,6 +549,23 @@ def describe(self):
             PipelineExecutionArn=self.arn,
         )
 
+    def display(self):
+        """Prints out a Graphviz DAG visual for a Pipeline's execution
+
+        Returns
+            A Graphviz object representing the execution, if Successful
+        """
+
+        pipelineGraph = PipelineGraph.from_pipeline(self.pipeline)
+        adjacencyList = pipelineGraph.build_adjacency_list_with_condition_edges()
+
+        step_statuses = {}
+        execution_steps = self.list_steps()
+        for step in execution_steps:
+            step_statuses[step["StepName"]] = step["StepStatus"]
+
+        return build_visual_dag(self.pipeline.name, adjacencyList, step_statuses)
```

Comment:
Same above, we can mock the list_steps and build_visual_dag to test the display method here